python-lib/gluonts_forecasts/custom_models/stl.py

```diff
@@ -0,0 +1,105 @@
+from gluonts.model.estimator import Estimator
+from gluonts.model.forecast import SampleForecast
+from gluonts.model.predictor import RepresentablePredictor
+from gluonts.support.pandas import frequency_add
+from gluonts.core.component import validated
+from gluonts_forecasts.custom_models.utils import cast_kwargs
+from constants import TIMESERIES_KEYS
+from statsmodels.tsa.api import STLForecast
+from statsmodels.tsa.exponential_smoothing.ets import ETSModel
+import numpy as np
+import pandas as pd
+from safe_logger import SafeLogger
+from tqdm import tqdm
+
+
+logger = SafeLogger("Forecast plugin - STL")
```

Comment:
```suggestion
logger = SafeLogger("Forecast plugin - Seasonal Trend")
```

Reply:
Calling it SeasonalTrend (no space) like AutoARIMA

74b2592